src/Game.js

```diff
@@ -0,0 +1,48 @@
+import { Component } from 'react';
+import React from 'react';
+import Board from './Board';
+import { revert, play } from './actions';
+import { connect } from 'react-redux';
+
+class Game extends Component {
+  render() {
+    const { history, winner, stepNumber, xIsNext } = this.props;
+    const current = history[stepNumber];
+    const status = winner ?
+      `Winner is ${winner}` :
+      `Next player: ${xIsNext ? 'X' : 'O'}`;
+
+    const moves = history.map((step, move) => {
+      const desc = move ?
+        'Go to move #' + move :
+        'Go to game start';
+      return (
+        <li key={move}>
+          <button onClick={() => this.props.onRevert(move)}>{desc}</button>
+        </li>
+      );
+    });
+
+    return (
+      <div className="game">
+        <div className="game-board">
+          <Board
+            squares={current} 
+            onClick={this.props.onPlay}/>
```

Comment:
This is a bit of a "small detail" kind of comment, but since you are learning react, is probably more important than usual:
The event on this case can have a more meaningful name, like `onSquareSelected` instead of `onClick`. The `onClick` event as such only happens for the `Square` component, on the `Board` component is not really a click (and actually its event data is not the event data corresponding a regular DOM click, but a more specific one that has its own data)
Again, this is a very small detail in general, but at least when starting is nice to do that as 1) it makes it less silly to do the usual passing down/up out events on the component hierarchy which leads to 2) sometimes it allows you to think about a better separation of concerns/abtractions between such components

Reply:
So true this is not a good name and yeah, I see this could be a spot of confusion when learning! Fortunately this case is just bad naming 😄 -- passing down event handlers is common in the Ember 🐹 world. I guess that being familiar with the concept, and the fact this is the name used in the react tutorial lead me to the laziness of leaving the `onClick` naming. 

I will change it. Thanks for all the explanation / context / dedication / looove!